mmv1/templates/terraform/examples/network_peering_routes_config_basic.tf.tmpl

```diff
@@ -2,8 +2,10 @@ resource "google_compute_network_peering_routes_config" "{{$.PrimaryResourceId}}
   peering = google_compute_network_peering.peering_primary.name
   network = google_compute_network.network_primary.name
 
-  import_custom_routes = true
-  export_custom_routes = true
+  import_custom_routes                = true
+  export_custom_routes                = true
+  import_subnet_routes_with_public_ip = true
+  export_subnet_routes_with_public_ip = true
 }
 
 resource "google_compute_network_peering" "peering_primary" {
```

Comment:
This will need to match the values above or we'll get an error- we're managing the same resource twice, which is a bit silly of an example (but is the easiest way to test the resource and presumably that's why we include it). Because we change the `google_compute_network_peering_routes_config` and that affects the `google_compute_network_peering` we're creating a diff.